web/package.json

```diff
@@ -145,8 +145,8 @@
     "prettier-fix": "prettier --write . && prettier --write ../config",
     "prettier-check": "prettier --check . && prettier --check ../config",
     "predevelop": "yarn generate-zones-config",
```

Comment:
Since we already have this part (which makes CI stop), I don't think we need to run the script on any commands - it is also very rarely changing :)

```js
  if (verifyNoUpdates) {
    console.error('Did not expect any updates to world.json. Please run "yarn update-world"');
    process.exit(1);
  }
```